src/EditorFeatures/Core/EmbeddedLanguages/AbstractEmbeddedLanguageEditorFeaturesProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.LanguageServices;
+using Microsoft.CodeAnalysis.Features.EmbeddedLanguages;
+using Microsoft.CodeAnalysis.Features.EmbeddedLanguages.RegularExpressions;
+
+namespace Microsoft.CodeAnalysis.Editor.EmbeddedLanguages
+{
+    /// <summary>
+    /// Abstract implementation of the C# and VB embedded language providers.
+    /// </summary>
+    internal abstract class AbstractEmbeddedLanguageEditorFeaturesProvider 
+        : AbstractEmbeddedLanguageFeaturesProvider, IEmbeddedLanguageEditorFeaturesProvider
+    {
+        new public ImmutableArray<IEmbeddedLanguageEditorFeatures> Languages { get; }
```

Comment:
>new [](start = 8, length = 3)

Why `new`, as opposed to an override, or maybe giving it a different name for clarity (if really different concept/API)?

Reply:
yup!